### PR TITLE
Update ReadMe

### PR DESCRIPTION
**Pull request by KomalBhadani27**
Updated readme to reflect new changes. Deployed version doesn't have animation option. It will be available after next deployment.

### CODE REVIEW

**Comment by telescopic** [comment]
Could you perhaps include a gif along with the readme ?

File: README.md [hunk 1]
```diff
@@ -55,16 +56,16 @@ variables:
...
 
   Example:
 
+    $ python main.py 'https://www.scoopwhoop.com/entertainment/memes-from-dd-ramayan/'
 
     $ python main.py 'https://www.scoopwhoop.com/entertainment/memes-from-dd-ramayan/' 5
 
-    $ python main.py 'https://www.scoopwhoop.com/entertainment/memes-from-dd-ramayan/'
-
+    $ python main.py 'https://www.scoopwhoop.com/entertainment/memes-from-dd-ramayan/' -enable_animations 'on'
```
Comment (detel) [inline README.md:68]: Its better to have boolean here, instead of `on`

**Review by detel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (KomalBhadani27) [reply README.md]: Changed the options to yes/no.

Comment (detel) [reply README.md]: Please check this ocne:
https://stackoverflow.com/questions/15008758/parsing-boolean-values-with-argparse

File: api/server.py [hunk 2]
```diff
@@ -51,6 +51,9 @@ def stampify_url():
     max_pages = request.args.get('max_pages')
     enable_animations = request.args.get('animations')
 
+    if not max_pages:
+        max_pages = 8
```
Comment (detel) [inline api/server.py:55]: Add comment here explaining that this is default value.

**Review by detel — APPROVED** [review]

File: api/server.py (hunk 2 above)
Comment (KomalBhadani27) [reply api/server.py:60]: Updated! Thanks for suggestion.

File: README.md (hunk 1 above)
Comment (KomalBhadani27) [reply README.md]: Updated accordingly!

**Pull request merged**